Containerfile.wasp

```diff
@@ -0,0 +1,34 @@
+# Builder stage
+FROM golang:1.20-alpine as builder
```

Comment:
Please use fedora as a base.

Reply:
Can you please elaborate why is it better?

Reply:
Because we are building for openshift, and openshift is based on rhel, and rhel on fedora.

Reply:
Has this been addressed?